src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -96,9 +98,32 @@ export const getAllThorchainSaversPositions = async (
     staleTime: 60_000,
   })
 
-  if (!opportunitiesData) return []
+  const { data: runepoolInformation } = await queryClient.fetchQuery({
+    queryKey: ['thorchainRunepoolInformation'],
+    queryFn: () =>
+      axios.get<ThorchainRunepoolInformationResponseSuccess>(
+        `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/runepool`,
+      ),
+    staleTime: 60_000,
+  })
+
+  const runepoolOpportunity: ThorchainSaverPositionResponse = {
+    asset: 'THOR.RUNE',
+    asset_address: fromAssetId(thorchainAssetId).assetReference,
+    last_add_height: undefined,
+    units: runepoolInformation.providers.units,
+    asset_deposit_value: runepoolInformation.providers.current_deposit,
+    asset_redeem_value: runepoolInformation.providers.value,
+    growth_pct: undefined,
+  }
```

Comment:
blocker: this is an opportunity, not a list of position (this method i.e `getAllThorchainSaversPositions` is fetching all addies and their position for a given saver.
It effectively return all active stakers for a given savers, so that we can iterate over them and find if UTXO accounts have an address active on them - we don't need this for runepool since THORChain is address-based.

Reply:
gm [f9f3a17](https://github.com/shapeshift/web/pull/7436/commits/f9f3a1724a6a028148e7c26bf70eea6b94c30fdc)